src/ontology/nbo-edit.owl

```diff
@@ -3240,10 +3242,13 @@ SubClassOf(obo:NBO_0000219 obo:NBO_0000210)
 
 # Class: obo:NBO_0000220 (social learning)
 
-AnnotationAssertion(obo:IAO_0000115 obo:NBO_0000220 "\"The sociological process of training individuals in a society to act or respond in a manner generally approved by the society in general and peer groups within society.\" [wikipedia:Social_conditioning]")
+AnnotationAssertion(obo:IAO_0000115 obo:NBO_0000220 "associative learning behavior that arises from observing or interacting with another individual")
```

Comment:
Source? And capital first letter and trailing period missing.

Reply:
This is a minor modification to solve a 2015 argument - we are not trying to academically source this definition yet - just point to our reasoning for the solution, but it is on the long list when we get round to reorganising the NBO.